packages/pigeon/platform_tests/android_kotlin_unit_tests/android/app/src/main/kotlin/com/example/android_kotlin_unit_tests/AllDatatypes.kt

```diff
@@ -0,0 +1,207 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
What are all these added files?  Were these added by mistake?

Reply:
That's really strange, these weren't in my local git info. I'll fix it.